code/game/machinery/atmoalter/gas_mine.dm

```diff
@@ -54,12 +54,10 @@
 	..()
 	if(anchored)
 		on=!on
-		update_icon()
-
-/obj/machinery/atmospherics/miner/attack_ai(var/mob/user)
```

Comment:
parent of attack_ai on machines calls attack_hand, zero reason to change the child here